backend/src/routes/api/notebooks/notebookUtils.ts

```diff
@@ -52,6 +52,16 @@ export const getNotebook = async (
   }
 };
 
+const checkPodContainersReady = (pod: V1Pod): boolean => {
+  const containerStatuses = pod.status?.containerStatuses || [];
```

Comment:
```suggestion
  const containerStatuses: V1ContainerStatus[] = pod.status?.containerStatuses || [];
```
When you do an `|| []` you mangle the types for TS... I just pulled the type of `containerStatuses` and put it here.